### ISSUE TITLE
Logits

### ISSUE THREAD

**Issue by Tongling-P** [pengr0925/DDloss#1]
How to get logits?

**Comment by pengr0925** [comment]
> How to get logits?
logits are the raw, unnormalized outputs of the last layer of a neural network before any activation function (like softmax or sigmoid) is applied.

**Comment by pengr0925** [comment]
> How to get logits?

logits are the raw, unnormalized outputs of the last layer of a neural network before any activation function (like softmax or sigmoid) is applied.

**Comment by Tongling-P** [comment]
Thank you for your answer

**Comment by Tongling-P** [comment]
Where should DDLoss be added? Should it be changed in maskrcnn_benchmark/modeling/roi_heads/relation_head/loss.py?

**Comment by Tongling-P** [comment]
How do I initialize the weight_init?
